ddtrace/internal/logger.py

```diff
@@ -172,7 +172,17 @@ def handle(self, record):
             self.buckets[key] = DDLogger.LoggingBucket(current_bucket, 0)
 
             # Call the base handle to actually log this record
-            super(DDLogger, self).handle(record)
+            c = self
+            found = 0
+            while c:
+                for hdlr in c.handlers:
+                    found = found + 1
+                    if record.levelno >= hdlr.level:
+                        hdlr.handle(record)
+                if not c.propagate or found >= 1:
+                    c = None  # type: ignore[assignment]
```

Comment:
nit: I would `break` here to make it explicit that we're bailing out to the loop.